bot.py

```diff
@@ -351,6 +353,31 @@ def __init__(self, *args, choices=None, opts_to_inv=None, **kwargs):
         )
         self.add_item(select_menu)
 
+class EmojiSyncView(discord.ui.View):
+    # Might be able to delete guild reference
+    def __init__(self, emoji, mod_type: str, old_name=None, *args, **kwargs):
+        super().__init__(timeout=None, *args, **kwargs)
+        self.emoji = emoji
+        self.old_name = old_name
+        
+        # Mod type will be a String of 'Add', 'Del', or 'Name' 
+        self.mod_type = mod_type
+
+    @discord.ui.button(label='Accept', style=discord.ButtonStyle.green)
```

Comment:
After interacting with these buttons you can continue pressing them - they should disappear once interacted with.

Reply:
This should be done. I did it so that the message will be edited to an ephemeral message that says "I will/will not sync this" so that the bot commands channel will not be spammed.